### DISCUSSION TITLE
Does nvim-cmp only work for neovim? or does it work for vim too?

### DISCUSSION THREAD

**Discussion by usrbinr** [hrsh7th/nvim-cmp#1702] [Q&A] [answered]
Hi 

Quick question -- does this plugin only work for neovim or can it work for vim too?

**Comment by gegoune** [comment]
It's written in lua using nvim's lua api, it will not work in vim.

**Comment by Shougo** [comment, marked as answer]
If you want to use Vim, you should use other completion plugins.
